.npmignore

```diff
@@ -6,4 +6,6 @@ bower_components
 examples
 report
 .idea
-*.html
\ No newline at end of file
+*.html
+.vscode
+dist
```

Comment:
Remove `dist` from `.npmignore`, or we ship nothing for the browser when doing `npm publish`

Reply:
fixed, thanks